src/Tools/Source/CompilerGeneratorTools/Source/BoundTreeGenerator/BoundNodeClassWriter.cs

```diff
@@ -996,24 +1012,25 @@ private void WriteUpdateMethod(Node node)
                         Outdent();
                         WriteLine("End Function");
                         break;
+
+                        string notEquals(Field field)
+                        {
+                            var parameterName = ToCamelCase(field.Name);
+                            var fieldName = field.Name;
+
+                            if (!IsValueType(field.Type))
+                                return $"{parameterName} IsNot Me.{fieldName}";
+
+                            if (GetGenericType(field.Type) == "OneOrMany")
+                                return $"Not {parameterName}.SequenceEqual({fieldName}, Symbols.SymbolEqualityComparer.ConsiderEverything)";
```

Comment:
>Symbols.SymbolEqualityComparer.ConsiderEverything [](http://example.com/codeflow?start=88&length=49)

What makes us think this comparer matches the ```<T>``` on ```OneOrMany```?

Reply:
I guess I can remove it. The default comparer `EqualityComparer<T>.Default` will work fine.